kart/conflicts.py

```diff
@@ -217,6 +216,10 @@ def conflicts_json_as_geojson(json_obj):
     """Converts the JSON output of list_conflicts to geojson."""
     features = []
     for key, feature in json_obj.items():
+        if type(feature) != dict or feature.get("type") != "Feature":
+            L.warning(f"Failed to convert meta-item conflicts to geojson.")
```

Comment:
Since converting anything that is not a geographic feature to geojson doesn't make sense, these errors could be more to the point eg "There are meta-item conflicts that cannot be output as geojson." ("Failed to convert" makes it seem like we attempted to convert them to geojson, but we don't).
This is a bit too warny right now - it warns about three times per conflict that you have, and each new warning adds no additional information. I would prefer any of the following:
- Warn once per conflict, include the conflict's key (but not the version part eg "ancestor/ours/theirs")
- Or, Warn once per dataset, include the name of the dataset.
- Or, just warn once if there are dropped meta-items at all.
Doing any of these will make the fix a little bit more complicated. But you can choose whichever one seems easiest or best to you.

No need to log the dropped meta items - if the user needs to see them, they can run a command that shows them.

Reply:
Thanks for the suggestion.